### ISSUE TITLE
ImportError: cannot import name 'image_data_format'

### ISSUE THREAD

**Issue by SherilynW** [keras-team/keras#8994]
It seems that there are something wrong with image_data_format.
```
ImportError                               Traceback (most recent call last)
<ipython-input-6-8e95bab5b83c> in <module>()
      1 from __future__ import print_function
      2 import tensorflow as tf
----> 3 import keras
      4 from keras.datasets import cifar10
      5 from keras.preprocessing.image import ImageDataGenerator

/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/keras/__init__.py in <module>()
      1 from __future__ import absolute_import
      2 
----> 3 from . import utils
      4 from . import activations
      5 from . import applications

/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/keras/utils/__init__.py in <module>()
      4 from . import data_utils
      5 from . import io_utils
----> 6 from . import conv_utils
      7 
      8 # Globally-importable utils.

/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/keras/utils/conv_utils.py in <module>()
      1 from six.moves import range
      2 import numpy as np
----> 3 from .. import backend as K
      4 
      5 

/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/keras/backend/__init__.py in <module>()
      9 from .common import set_floatx
     10 from .common import cast_to_floatx
---> 11 from .common import image_data_format
     12 from .common import set_image_data_format
     13 

ImportError: cannot import name 'image_data_format'
```

**Comment by SherilynW** [comment]
This error happens just at the beginning when I try to import keras.

**Comment by SherilynW** [comment]
Bug fixed Pull#8995
